### PR TITLE
fix(component PluginButton): Additional ... in RTE throws type is invalid error and serves WSOD (#536)

### LINKED ISSUE #536: Additional ... in RTE throws type is invalid error and serves WSOD
## Description

Additional Styles doesn't work in RTE

### Steps to reproduce

1. Configure the RTE to serve additional styles so you get the ... (Additional Styles button)
2. Edit src/components/Editors/index.tsx#L81 
3. add sample styles:
```
  CenterHeader: flow(asHeader1, asAlignCenter, asItalic),
  UnderlineRightHeader: flow(asHeader1, asAlignRight, asUnderline),
```
4. Bring up Site, go to RTE field that uses full editor, be in edit mode.
5. Click the ... in Rich Text Editor Toolbar

Can be seen on https://edit.master-7rqtwti-gc5fzrr5cpumq.us-2.platformsh.site/articles/articleyoutube

### Expected result

Shows additional Styles

### Actual result

https://www.screencast.com/t/p6z0u0i9z

WSOD
+ 
console errors

```
index.js:2177 Warning: React.createElement: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check your code at PluginButton.tsx:49.
    in PluginButton (at createPluginButton.tsx:48)
    in Unknown (at RichTextItemGetters.tsx:264)
    in Unknown (at RichText.tsx:205)
    in div (at FClasses.tsx:128)
    in Stylable (at FClasses.tsx:51)
    in AddClasses (at TextSelectorButton.tsx:72)
    in div (created by Content)
    in Content (created by Trigger)
    in div (created by LazyRenderBox)
    in LazyRenderBox (created by PopupInner)
    in div (created by PopupInner)
    in PopupInner (created by Popup)
    in Align (created by Popup)
    in AnimateChild (created by Animate)
    in Animate (created by Popup)
    in div (created by Popup)
    in Popup (created by Trigger)
    in Portal (created by Trigger)
    in Trigger (created by Tooltip)
    in Tooltip (at TextSelectorButton.tsx:64)
    in TextSelectorButton (at RichText.tsx:222)
    in div (at FClasses.tsx:128)
    in Stylable (at hoc-util.tsx:44)
    in Unknown (at FClasses.tsx:51)
    in AddClasses (at HoverMenu.tsx:92)
    in HoverMenu (at HoverMenu.tsx:36)
    in StyledHoverMenu (at RichText.tsx:180)
    in EditOnlyHoverMenu$ (at RichText.tsx:215)
    in Unknown (at hoc.tsx:30)
    in WithoutProps (at withFlowToggle.tsx:32)
    in wrappedComponent (at SlateEditor.tsx:75)
    in Unknown (at withNodeStateHandlers.tsx:51)
    in wrappedComponent (at withNode.tsx:30)
    in NodeProvider (at withNode.tsx:29)
    in WithNode (at RichText.tsx:208)
    in BasicRichText (at hoc-util.tsx:35)
    in Unknown (at Design.tsx:188)
    in Transformer (at Design.tsx:204)
    in Unknown (at RichText.tsx:39)
    in RichText (at Design.tsx:154)
    in WithDesign (at richtext/index.tsx:47)
    in div (at richtext/index.tsx:46)
    in div (at richtext/index.tsx:44)
    in div (at FClasses.tsx:128)
    in Stylable (at FClasses.tsx:51)
    in AddClasses (at FClasses.tsx:51)
    in AddClasses (at Layout/index.tsx:38)
    in Layout (at richtext/index.tsx:28)
    in div (at FClasses.tsx:128)
    in Stylable (at hoc-util.tsx:44)
    in Unknown (at FClasses.tsx:51)
    in AddClasses (at FClasses.tsx:51)
    in AddClasses (at FClasses.tsx:51)
    in RemoveClasses (at ContextWrapper.tsx:38)
    in ContextWrapper (at ContextWrapper.tsx:46)
    in ContextWrapper (at Page.tsx:73)
    in PageEditor (at PageEditor.tsx:27)
    in PageEditor (at Page.tsx:70)
    in wrappedComponent (at Page.tsx:68)
    in GatsbyPageProvider (at Page.tsx:67)
    in NodeProvider (at GatsbyNodeProvider.tsx:75)
    in NodeProvider (at GatsbyNodeProvider.tsx:74)
    in GatsbyNodeProvider (at Page.tsx:66)
    in wrappedComponent (at richtext/index.tsx:27)
    in RichTextPage (created by HotExportedRichTextPage)
    in AppContainer (created by HotExportedRichTextPage)
    in HotExportedRichTextPage (created by PageRenderer)
    in PageRenderer (at query-result-store.js:86)
    in PageQueryStore (at root.js:56)
    in RouteHandler (at root.js:78)
    in div (created by FocusHandlerImpl)
    in FocusHandlerImpl (created by Context.Consumer)
    in FocusHandler (created by RouterImpl)
    in RouterImpl (created by Context.Consumer)
    in Location (created by Context.Consumer)
    in Router (at root.js:73)
    in ScrollHandler (at root.js:69)
    in RouteUpdates (at root.js:68)
    in EnsureResources (at root.js:66)
    in LocationHandler (at root.js:124)
    in LocationProvider (created by Context.Consumer)
    in Location (at root.js:123)
    in Root (at root.js:132)
    in StaticQueryStore (at root.js:138)
    in _default (at app.js:103)
__stack_frame_overlay_proxy_console__ @ index.js:2177
overrideMethod @ react_devtools_backend.js:2273
printWarning @ react.development.js:315
error @ react.development.js:287
createElementWithValidation @ react.development.js:1788
React$$1.createElement @ react-hot-loader.development.js:2875
PluginButton @ PluginButton.tsx:48
renderWithHooks @ react-dom.development.js:14804
mountIndeterminateComponent @ react-dom.development.js:17483
beginWork @ react-dom.development.js:18597
beginWork$1 @ react-dom.development.js:23180
performUnitOfWork @ react-dom.development.js:22155
workLoopSync @ react-dom.development.js:22131
performSyncWorkOnRoot @ react-dom.development.js:21757
(anonymous) @ react-dom.development.js:11090
unstable_runWithPriority @ scheduler.development.js:653
runWithPriority$1 @ react-dom.development.js:11040
flushSyncCallbackQueueImpl @ react-dom.development.js:11085
flushSyncCallbackQueue @ react-dom.development.js:11073
discreteUpdates$1 @ react-dom.development.js:21894
discreteUpdates @ react-dom.development.js:807
dispatchDiscreteEvent @ react-dom.development.js:4169

```

### Environment

  System:
    OS: macOS 10.15.6
    CPU: (4) x64 Intel(R) Core(TM) i3-8100B CPU @ 3.60GHz
    Shell: 5.7.1 - /bin/zsh
  Binaries:
    Node: 12.18.1 - ~/.nvm/versions/node/v12.18.1/bin/node
    npm: 6.14.5 - ~/.nvm/versions/node/v12.18.1/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 85.0.4183.83
    Safari: 13.1.2
  npmGlobalPackages:
    gatsby-cli: 2.12.60

### Affected Version

- v0.0.55

### PR DESCRIPTION
**Pull request by marcopagliarulo**
## Changes
Fix the WSOD when additional styles are are added to the Rich Text Editor.

## Test Instructions
Steps to reproduce

1. Configure the RTE to serve additional styles so you get the ... (Additional Styles button)
2. Edit https://github.com/johnsonandjohnson/Bodiless-JS/blob/master/examples/test-site/src/components/Editors/index.tsx#L81
3. add sample styles:
   CenterHeader: flow(asHeader1, asAlignCenter, asItalic),
   UnderlineRightHeader: flow(asHeader1, asAlignRight, asUnderline),
4. Bring up Site, go to RTE field that uses full editor, be in edit mode.
5. Click the ... in Rich Text Editor Toolbar
Expected result
The additional styles are shown without error on Console
## Related Issues
Fixes #536

### CODE REVIEW

**Review by VancheeZze — DISMISSED** [review]

File: examples/test-site/src/components/Editors/index.tsx [hunk 1]
```diff
@@ -79,6 +79,8 @@ const fullFeaturedDesign = {
   H2: asHeader2,
   H3: asHeader3,
   BlockQuote: flow(replaceWith(Blockquote), asBlockQuote, withQuoteBlockMeta),
+  CenterHeader: flow(asHeader1, asAlignCenter, asItalic),
```
Comment (hvanyo) [inline examples/test-site/src/components/Editors/index.tsx:82]: Lets call it CenterItalicHeader for all 3 tokens

File: packages/bodiless-richtext/src/components/TextSelectorButton.tsx [hunk 2]
```diff
@@ -65,14 +65,19 @@ const TextSelectorButton = ({
...
+      }}
       overlay={() => (
 
         // error TS2604: JSX element type 'Overlay' does not have any construct or call signatures.
         // @ts-ignore
         <Overlay>
           <TextSelectorContext.Provider value={textSelectorContextValue}>
             <CloseBtn onMouseDown={() => setVisible(false)} />
-            { children }
+            <div className="grid bg-white rounded p-2">
```
Comment (hvanyo) [inline packages/bodiless-richtext/src/components/TextSelectorButton.tsx:78]: per https://edit.master-7rqtwti-jvaff4bu65vgm.us-2.platformsh.site/___docs/#/Development/Architecture/AdminUI?id=bodiless-classes
please preface this to use -bl

We try put most styling to https://github.com/johnsonandjohnson/Bodiless-JS/tree/master/packages/bodiless-components-ui but not sure if its correct in this case.  @VancheeZze  defer to you on this.

File: examples/test-site/src/components/Editors/index.tsx (hunk 1 above)
Comment (hvanyo) [reply examples/test-site/src/components/Editors/index.tsx:82]: @marcopagliarulo 
Testing-- as align center/right need to be done on block/paragraph level and when tested it currently they don't center /right align and only applies to chars.

need to
import {
  asBlock,
} from '@bodiless/richtext';

and add that to each style.

and I see we never defined asItalic() -- can you add italic to in elements-token.ts + define asBold() as well :) font-bold while you are in there.  

If you are willing could you also delete these two extraneous files -- I guarantee they are not used..
examples/test-site/src/components/Editors/items.tsx
examples/starter/src/components/Editors/items.tsx

Comment (marcopagliarulo) [reply examples/test-site/src/components/Editors/index.tsx]: Everything fixed, thanks.

File: packages/bodiless-richtext/src/components/TextSelectorButton.tsx (hunk 2 above)
Comment (marcopagliarulo) [reply packages/bodiless-richtext/src/components/TextSelectorButton.tsx]: Everything fixed, thanks.

**Review by VancheeZze — APPROVED** [review]

**Pull request merged**